src/lib/utils/pdf-generator.ts

```
@@ -0,0 +1,261 @@
+				const section = sections[i];
+				
+				// Check if we need a new page
+				if (currentY > pageHeight - margin) {
+					pdf.addPage();
+					currentY = margin; // Same top margin as bottom margin
+				}
+
+				// Handle different content types
+				if (section instanceof HTMLHeadingElement) {
+					pdf.setFont('helvetica', 'bold');
+					let fontSize, marginTop, marginBottom;
```

Biome (1.9.4)

[error] Declare variables separately

Unsafe fix: Break out into multiple declarations


(lint/style/useSingleVarDeclarator)